src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource_Async.cs

```diff
@@ -125,7 +126,7 @@ private partial class SuggestedActionsSource : IAsyncSuggestedActionsSource
                                 var allSets = GetCodeFixesAndRefactoringsAsync(
                                     state, requestedActionCategories, document,
                                     range, selection,
-                                    new SuggestedActionPriorityProvider(priority, lowPriorityAnalyzers),
+                                    new SuggestedActionPriorityProvider(priority, lowPriorityAnalyzers, lowPriorityAnalyzerSupportedDiagnosticIds),
```

Comment:
same question.  it's interesting that it is captured by SuggestedActionPriorityProvider, so presumably writen to.  but where it is read isn't clear to me yet.